lib/index.js

```diff
@@ -101,6 +101,7 @@ class Client {
    * Get repos
    * @param {integer} page Page number
    * @param {integer} limit Page limit
+   * Ref: https://github.com/harness/drone/blob/v2.11.1/handler/api/repos/all.go#L30
```

Comment:
With this ref and the ref below should we be referring to a specific version of Drone? - the version number (and code) changes pretty regularly.

Perhaps https://github.com/harness/drone/blob/master/handler/api/repos/all.go and a reference to the method name would be better? I would suggest line number, but these are liable to change :(

Reply:
My thoughts were that if the version changes and a file is moved, then I'll be referencing master will give a dead link. 

It will also be hard to know what was supposed to be supported. 

If I reference the version gives us a concrete point in time, it should be easy to look at the changsets and either find the new endpoint, or see if any params have changed.

Reply:
ok, fair enough :)